lib/crate_ruby/client.rb

```diff
@@ -83,13 +85,13 @@ def drop_table(table_name, blob=false)
     # List all user tables
     # @return [ResultSet]
     def show_tables
-      execute("select table_name from information_schema.tables where table_schema = 'doc'")
+      execute("select table_name from information_schema.tables where table_schema = '#{schema}'")
```

Comment:
use parameter substitution

Reply:
possible, but not rubyish to do so. will therefore keep as is

Reply:
misunderstood, working on it.